packages/frontend/src/hooks/useSwapsData.ts

```diff
@@ -47,107 +47,82 @@ export const useSwapsData = () => {
   }, [address, shortHelper, squeethPool, subscribeToMore, swapRouter])
   const swaps = data?.swaps
   const isWethToken0 = parseInt(weth, 16) < parseInt(oSqueeth, 16)
-  const {
-    squeethAmount,
-    wethAmount,
-    longRealizedSqueeth,
-    totalUSDSpent,
-    longRealizedUSD,
-    longTotalSqueeth,
-    shortRealizedSqueeth,
-    shortRealizedUSD,
-    totalUSDReceived,
-    shortTotalSqueeth,
-    longUsdAmount,
-    shortUsdAmount,
-  } = useMemo(
-    () =>
-      swaps?.reduce(
-        (acc, s) => {
-          //values are all from the pool pov
-          //if >0 for the pool, user gave some squeeth to the pool, meaning selling the squeeth
-          const squeethAmt = new BigNumber(isWethToken0 ? s.amount1 : s.amount0)
-          const wethAmt = new BigNumber(isWethToken0 ? s.amount0 : s.amount1)
-          const usdAmt = getUsdAmt(wethAmt, s.timestamp)
+  const { squeethAmount, wethAmount, totalUSDFromBuy, boughtSqueeth, totalUSDFromSell, soldSqueeth, shortUsdAmount } =
+    useMemo(
+      () =>
+        swaps?.reduce(
+          (acc, s) => {
+            //values are all from the pool pov
+            //if >0 for the pool, user gave some squeeth to the pool, meaning selling the squeeth
+            const squeethAmt = new BigNumber(isWethToken0 ? s.amount1 : s.amount0)
+            const wethAmt = new BigNumber(isWethToken0 ? s.amount0 : s.amount1)
+            const usdAmt = getUsdAmt(wethAmt, s.timestamp)
 
-          //buy one squeeth means -1 to the pool, +1 to the user
-          acc.squeethAmount = acc.squeethAmount.plus(squeethAmt.negated())
-          acc.wethAmount = acc.wethAmount.plus(wethAmt.negated())
+            //buy one squeeth means -1 to the pool, +1 to the user
+            acc.squeethAmount = acc.squeethAmount.plus(squeethAmt.negated())
 
-          //<0 means, buying squeeth
-          //>0 means selling squeeth
+            //<0 means, buying squeeth
+            //>0 means selling squeeth
+            if (squeethAmt.isPositive()) {
```

Comment:
hmm this looks like stuff from the dollar pnl pr? just want to make sure we don't merge dollar pnl stuff in with the positions hooks refactor because that isn't ready quite yet until we figure out the API

Reply:
Yes, I updated this here too because it also had the bug where realized pnl was being calculated with no withdrawal after I removed the useLongPosition and useShortPosition hooks